### PR TITLE
Bump Mobius dependency version

### PR DESCRIPTION
**Pull request by msasikanth**

### CODE REVIEW

File: app/src/main/java/org/simple/clinic/allpatientsinfacility/AllPatientsInFacilityEffectHandler.kt [hunk 1]
```diff
@@ -8,24 +8,23 @@ import org.simple.clinic.facility.FacilityRepository
 import org.simple.clinic.patient.PatientRepository
 import org.simple.clinic.patient.PatientSearchResult
 import org.simple.clinic.util.scheduler.SchedulersProvider
+import javax.inject.Inject
```
Comment (Honey14) [inline app/src/main/java/org/simple/clinic/allpatientsinfacility/AllPatientsInFacilityEffectHandler.kt:11]: We did not need this commit in the current PR. We should've created a different PR for maintaining this. Merging this now.

File: buildSrc/src/main/kotlin/Versions.kt [hunk 2]
```diff
@@ -54,7 +54,7 @@ object Versions {
   const val playCore = "1.10.0"
   const val fastScroll = "2.0.0"
   const val mixpanel = "5.6.4"
-  const val mobius = "1.3.4"
+  const val mobius = "1.5.3"
```
Comment (Honey14) [inline buildSrc/src/main/kotlin/Versions.kt:57]: Was changing the version twice approach to make sure there are no major changes in the versions and nothing is broke?

**Review by Honey14 — APPROVED** [review]
Looks good. Left 2 questions

File: buildSrc/src/main/kotlin/Versions.kt (hunk 2 above)
Comment (msasikanth) [reply buildSrc/src/main/kotlin/Versions.kt:57]: Yep

File: app/src/main/java/org/simple/clinic/allpatientsinfacility/AllPatientsInFacilityEffectHandler.kt (hunk 1 above)
Comment (msasikanth) [reply app/src/main/java/org/simple/clinic/allpatientsinfacility/AllPatientsInFacilityEffectHandler.kt:11]: Actually, we had for this 😅 . With Mobius update, the older approach was not working for this, so I had to change it to our latest implementations.

**Review by Honey14 — APPROVED** [review]

**Pull request merged**